kinetic/distribution.yaml

```diff
@@ -0,0 +1,14 @@
+%YAML 1.1
+# ROS distribution file
+# see REP 143: http://ros.org/reps/rep-0143.html
+---
+release_platforms:
+  fedora:
+  - '22'
+  - '23'
```

Comment:
We need to be more aggressive, I think. We were off with Jade, which should have supported f23, which is the current release of Fedora.

I'm thinking 23/24/25 would be closer. However, if patterns hold, f26 would be dropping around June of 2017, which might mean that it falls before ROS-L releases, meaning that f26 wouldn't be supported on any ROS distros for some amount of time.

Going with 23/24/25/26 would be safer, but I think doubling the amount of RPM branches generated in Bloom would get really time consuming.

It would be interesting to look into a "shortcut" in Bloom for detecting when the generated branches will be exactly the same based on the resolved dependencies (which is REALLY common for RPM .spec files) and not generate those branches again...but I digress.

Thoughts?